Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -62,15 +62,14 @@ Otherwise, the first variant matching the URL parameters is used for generating
 and resolving the route.
 
 
-.. index:: Routing; Enhancers
+.. index:: Routing; Route Enhancers
 
 .. _routing-advanced-routing-configuration-enhancers:
 
-Enhancers
-=========
+Route Enhancers
+===============
 
-There are two types of enhancers: decorators and route enhancers. A route
```

Comment:
I think this sentence has important info and should be kept to avoid confusion below.  
Furthermore  it might be useful to give `aspect` a logical context, if it's only used in route enhancers or if it's independent and could be used in `decorators` too. Clarifying that `aspect` and `decorator` are not the same could also help, because the word `aspect` in the text is used out of nowhere whereas decorators and route enhancers are introduced by the sentence above.

Reply:
But this sentence seems to be wrong and causes confusion.

Reply:
Please see the list below (line 79 to 9.1), a route enhancer and a decorator are not the same thing.